Authenticator/Source/AppController.swift

```diff
@@ -43,8 +43,15 @@ class AppController {
     }()
 
     init() {
+        if Process.isDemo {
+            store = DemoTokenStore()
```

Comment:
Code for reference https://github.com/beaucollins/Authenticator/commit/b689014fb8d53cb97e1f2d67b30f29c0a677fd26